codeship-services.yml

```diff
@@ -1,5 +1,5 @@
 dynamodb:
-  image: amazon/dynamodb-local
+  image: amazon/dynamodb-local:1.22.0
```

Comment:
The image we were previously using was deprecated:

<img width="867" alt="image" src="https://github.com/ginger-io/ginger-vault-service/assets/1993853/39c5a13c-da83-41c5-b50e-8908d540045a">